demos/code/code.js

```diff
@@ -304,45 +306,63 @@ Code.renderContent = function() {
     xmlTextarea.value = xmlText;
     xmlTextarea.focus();
   } else if (content.id == 'content_javascript') {
-    var code = Blockly.JavaScript.workspaceToCode(Code.workspace);
-    content.textContent = code;
-    if (typeof PR.prettyPrintOne == 'function') {
-      code = content.textContent;
-      code = PR.prettyPrintOne(code, 'js');
-      content.innerHTML = code;
-    }
+    try{
```

Comment:
Instead of using several smaller `try` statements inside each `if` block, I think it makes sense to have one large `try` block encompassing all the `if` blocks.